workspaces/ui/src/routes.js

```diff
@@ -41,7 +41,7 @@ export function ApiRoutes(props) {
             />
           )}
 
-        <Redirect to={routerPaths.docsRoot} />
+        <Redirect to={props.demo ? routerPaths.diffsRoot : routerPaths.docsRoot} />
```

Comment:
Conceptually, I'd prefer the generic routes to not have knowledge of things like debugging and demos. The pathways for having different behaviors between each entry point should be injecting things at the entry point and factoring extension points of the UI into the shared tree and providing components for those slots. When does this pathway get hit?

Reply:
This gets hit from `demo-sessions.js` https://github.com/opticdev/optic/pull/290/files#diff-0c55cfef92a59cc08e309f67f2b0bae8R230

agree that there should be a better way — I was looking for a way to simply specify a default page - I think a general prop called "default" could be good to include, considering we may want to specify a default page (or have the ui remember the users last page)

Reply:
Yeah I think adding a defaultRoute prop set to docs by default would make sense